MapleServer2/Packets/StatPacket.cs

```diff
@@ -18,7 +19,18 @@ public static Packet SetStats(IFieldObject<Player> player)
             pWriter.WriteInt(player.ObjectId);
             pWriter.WriteByte();
             pWriter.WriteEnum(StatsMode.SendStats);
-            pWriter.Write(player.Value.Stats);
+            WriteStats(ref pWriter, player.Value.Stats);
+
+            return pWriter;
+        }
+
+        public static Packet UpdateStats(IFieldObject<Player> player)
+        {
+            PacketWriter pWriter = PacketWriter.Of(SendOp.STAT);
+            pWriter.WriteInt(player.ObjectId);
+            pWriter.WriteByte();                // Unknown (0x00/0x01)
+            pWriter.WriteByte(0x23);            // Unknown
```

Comment:
Instead of hardcoding, use the enum you already have.
```suggestion
            pWriter.WriteEnum(StatsMode.SendStats);
```